db/migrate/20170502194830_create_invoices.rb

```diff
@@ -0,0 +1,11 @@
+class CreateInvoices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :invoices do |t|
+      t.string :status
+      t.datetime :created_at
+      t.datetime :updated_at
+
+      t.timestamps
```

Comment:
Think it would be clearer if we render the timestamps as null: true since we already have created_at and updated_at in our model.